src/PhpSpreadsheet/Cell/Cell.php

```diff
@@ -139,7 +139,7 @@ public function getRow()
      */
     public function getCoordinate()
     {
-        return $this->parent->getCurrentCoordinate();
+        return $this->parent->getCurrentCoordinate() ?? '';
```

Comment:
I'd probably go for an exception then. If the conditions to see this case is as degenerate as you mention, then it probably is best to inform the user that something wrong already happened, instead of pretending that things are fine.

Reply:
> I am very glad to see you back!

And I am very glad you are still around. I am impressed by the work did in this project. Thanks a lot !

If anything I'd say you don't have to bend to the whims to Scrutinizer, as mentioned in my next comment. It is a very good guideline to follow, but sometimes Scrutinizer is just plain wrong (unfortunately).

Also if your PR fixes an existing issue, you can include it in the commit message, such as `Fixes #123`, so that when the PR is merged, the issue is automatically closed at the same time. See https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword. Or alternatively you can close the issue manually yourself whenever you're quite sure your PR will get merged.

On my side, I'll try to release a bit more often than in the past months. And I'd like to move some (but not all) PHPdocs to proper type hints.

Reply:
Changed to use exception, added some tests - explicitly deleting the cell is another situation in which this will arise.

Reply:
I will try to remember `Fixes #123` in future. I would prefer that the originator close the ticket, but I've noticed that doesn't happen all that often.